packages/client/src/features/tools/change-bounds/change-bounds-tool.ts

```diff
@@ -183,7 +167,7 @@ export class ChangeBoundsListener extends DragAwareMouseListener implements ISel
                 const resizeActions = this.handleResizeOnClient(positionUpdate);
                 actions.push(...resizeActions);
             }
-            this.tool.dispatchFeedback(actions, this);
+            return actions;
```

Comment:
Doesn't this undo the [earlier fix](https://github.com/eclipse-glsp/glsp-client/pull/262) showing moves as feedback (and thus re-apply on `UpdateModelAction`?

Reply:
Maybe I merged something incorrectly, will double check! I tested some aspects but maybe not all.

Reply:
Yeah, it was a bad merge, I'll restore the old behavior.